roles/baseline/defaults/main.yml

```diff
@@ -25,12 +25,25 @@ METRICS_SERVER_CONFIG:
   apiService:
     create: true
 
+## Ingress-nginx - Defaults
+ingressVersions:
+  k8sMinorVersionCeiling:
+    value: 21
+    api:
+      chartVersion: 3.20.1
+      appVersion: 0.43.0
+  k8sMinorVersionFloor:
+    value: 22
+    api:
+      chartVersion: 4.0.13
+      appVersion: 1.1.0
+
 ## Ingress-nginx - Ingress
 INGRESS_NGINX_NAME: ingress-nginx
 INGRESS_NGINX_NAMESPACE: ingress-nginx
 INGRESS_NGINX_CHART_NAME: ingress-nginx
 INGRESS_NGINX_CHART_URL: https://kubernetes.github.io/ingress-nginx
-INGRESS_NGINX_CHART_VERSION: 3.20.1
+INGRESS_NGINX_CHART_VERSION: UNSPECIFIED
```

Comment:
Our coding practices in the past have been to set items that were dynamically decided with either the empty string `""` , `null` or just empty like `INGRESS_NGINX_CHART_VERSION: `. Not sure why UNSPECIFIED was decided upon here. We have not in the past used this type of login. If moving forward this is the right way for the team, be aware, there are several areas in the code base that `""` empty strings and null are being used.

Reply:
In my latest patch set, updated the default value to adhere to the project conventions. `INGRESS_NGINX_CHART_VERSION` default value is now an empty string. Ready for re-review.